backend/tests/unit/utils/test_similarity_functions.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from oasst_backend.utils.similarity_functions import cosine_similarity, euclidean_distance
+
+
+def test_cosine_similarity():
```

Comment:
:'( .. I think https://github.com/LAION-AI/Open-Assistant/pull/1650 wants to remove those functions...

Reply:
It's okay, `copilot` wrote the tests for these ones :)

Should I remove them, or keep and remove the tests afterwards?

Reply:
I can put them back in!

Reply:
I think that if they're code that isn't used then it's best to remove them. Think of code as "lines spent" rather than "lines produced"

Reply:
Cool I'll update this later on, to cover the new functions :)

Reply:
done :)